### DISCUSSION TITLE
What is the Red Attention icon on my Servers menu item?

### DISCUSSION THREAD

**Discussion by bjazmoore** [GetPublii/Publii#1651] [Help] [answered]
After configuring a server for GitHub server connection, and successfully connecting I saved the settings.  Then I noticed the red attention icon on the Servers menu item as shown in this image:

![Figure 8-35](https://github.com/user-attachments/assets/742c442d-0b31-48aa-a2a2-6ce00429e5c0)

Just curious what this means.  I cannot find any errors and I was able to sync my site to GitHub just fine.

**Comment by bobmitro** [comment, marked as answer]
This icon is displayed because, in version 0.44, we started the process of phasing out support for GitHub and GitLab, with plans to remove these platforms in future versions. However, based on community feedback, we decided to keep these options. Although we removed the deprecation messages, we missed removing the icon from the menu. You don't need to worry - the icon will be removed in the next app release.
